File: src/main/java/no/nav/personopplysninger/features/endreopplysninger/domain/adresse/EndringGateadresse.java
package no.nav.personopplysninger.features.endreopplysninger.domain.adresse;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import no.nav.personopplysninger.features.endreopplysninger.domain.Endring;

@JsonIgnoreProperties(ignoreUnknown = true)
public class EndringGateadresse extends Endring<EndringGateadresse> {
    private Gateadresse innmeldtEndring;

    public Gateadresse getInnmeldtEndring() {
        return innmeldtEndring;
    }
}

File: src/main/kotlin/no/nav/personopplysninger/features/endreopplysninger/domain/adresse/EndringGateadresse.kt
package no.nav.personopplysninger.features.endreopplysninger.domain.adresse

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import no.nav.personopplysninger.features.endreopplysninger.domain.Endring

@JsonIgnoreProperties(ignoreUnknown = true)
class EndringGateadresse : Endring<EndringGateadresse>() {
    val innmeldtEndring: Gateadresse? = null
}
